changelog.json

```diff
@@ -1,5 +1,27 @@
 {
   "releases": {
+    "2.1.2": [
+      "[Added] Syntax highlighting support for 20 more programming languages - #7217. Thanks @KennethSweezy!",
+      "[Added] Kitty shell support for macOS - #5162",
+      "[Added] Atom editor support on Windows for beta and nightly channels - #8010. Thanks @Rexogamer!",
+      "[Added] WSL Support - #7821 #7891. Thanks @ADustyOldMuffin @say25!",
+      "[Fixed] File checkboxes and line selection in diffs are not disabled when committing - #3814 #5934. Thanks @HashimotoYT!",
+      "[Fixed] Small images are scaled down too much in two-up image rendering - #7520",
+      "[Fixed] Manual conflict resolution for binary files always chooses same version - #8059",
+      "[Fixed] Disable some unavailable actions while committing - #3814 #5934. Thanks @HashimotoYT!",
+      "[Fixed] Branch pruner errors on \"orphaned\" branches - #7983",
+      "[Fixed] Prevent CodeMirror search plugin from stealing registered global keyboard shortcuts - #8068",
+      "[Fixed] Update embedded Git on Windows to address security vulnerability - #8101, #8133",
```

Comment:
```suggestion
      "[Fixed] Update embedded Git on Windows to remove erroneous errors - #8101, #8133",
```

Reply:
Ah, you're right. And we should drop the #8101 link too.